pretext/ExtraTopics/Glossary.ptx

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-glossary">
+  <title>Glossary</title>
+
+  
+  <!-- BOILERPLATE STUFF -->
+  <glossary sorted="False">
+    <gi>
+      <title>activecode</title>
+      <p>A unique interpreter environment that allows Python to be executed from within a web browser.</p>
+    </gi>
+    <gi>
+      <title>algorithm</title>
+      <p>A general step by step process for solving a problem.</p>
+    </gi>
+    <gi>
+      <title>bug</title>
+      <p>An error in a program.</p>
+    </gi>
+    <gi>
+      <title>byte code</title>
+      <p>An intermediate language between source code and object code. Many
```

Comment:
This is way to technical for a 100 level class. Object code also isn't a thing in python. It goes source -> AST -> Byte code -> execution for python. Tho it doesn't even need the byte code, python can directly execute the AST.